src/commands/createStaticWebApp/setWorkspaceContexts.ts

```diff
@@ -0,0 +1,18 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as fse from 'fs-extra';
+import { join } from "path";
+import { IActionContext } from "vscode-azureextensionui";
+import { remoteShortnameExists } from "../../utils/gitHubUtils";
+import { IStaticWebAppWizardContext } from "./IStaticWebAppWizardContext";
+
+export async function setWorkspaceContexts(wizardContext: IActionContext & Partial<IStaticWebAppWizardContext>, fsPath: string): Promise<void> {
```

Comment:
This func is only used in `createStaticWebApp.ts`, any reason you didn't put it there?

Reply:
Yeah, I prefer just giving commands their own file rather than cluttering files with helper functions.  It also used to be part of `WorkspaceListStep` so I just thought it'd be better as a standalone command and in the same folder (`createStaticWebApp`) as the other create helper files.